sunpy/visualization/draw.py

```diff
@@ -0,0 +1,106 @@
+"""
+This module provides functions that draw on Map plots.
+"""
+import numpy as np
+from matplotlib.lines import Line2D
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.coordinates import HeliographicStonyhurst
+from sunpy.visualization import wcsaxes_compat
+
+__all__ = ["draw_equator", "draw_prime_meridian"]
+
+
+def draw_equator(axes, resolution=300, **kwargs):
+    """
+    Draws the equator as seen by the axes observer.
+
+    Parameters
+    ----------
+    axes : `matplotlib.axes.Axes`
+        The axes to plot the equator on.
+    resolution : `int`
+        The number of points used to represent the equator.
+
+    Examples
+    --------
+    >>> import sunpy.map
+    >>> from sunpy.visualization import draw
+    >>> import sunpy.data.sample   # doctest: +REMOTE_DATA
+    >>> aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)   # doctest: +REMOTE_DATA
+    >>> aia.plot()   # doctest: +SKIP
+    >>> draw.draw_equator(aia._check_axes(None))   # doctest: +SKIP
+    """
+    if not wcsaxes_compat.is_wcsaxes(axes):
+        raise ValueError('axes must be a WCSAxes')
+    axes_frame = axes._transform_pixel2world.frame_out
```

Comment:
We can avoid using this private attribute by doing the following:
```suggestion
    from astropy.visualization.wcsaxes.wcsapi import wcsapi_to_celestial_frame
    axes_frame = wcsapi_to_celestial_frame(axes.wcs)
```